include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -664,396 +659,196 @@ namespace picongpu
 
-                    for(uint32_t detectorDim = 0; detectorDim < 3; ++detectorDim)
-                    {
-                        splash::Dimensions offset(detectorDim, 0, 0);
-                        splash::Selection dataSelection(
-                            bufferSizeDetector,
-                            componentSizeDetector,
-                            offset,
-                            strideDetector);
-
-                        hdf5DataFile.write(
-                            currentStep,
-                            radSplashType,
-                            3,
-                            dataSelection,
-                            (meshesPathName + dataLabelsDetectorDirection(detectorDim)).c_str(),
-                            detectorPositions.data());
-
-                        /* save SI unit as attribute together with data set */
-                        const picongpu::float_64 factorDirection = 1.0;
-                        hdf5DataFile.writeAttribute(
-                            currentStep,
-                            radSplashType,
-                            (meshesPathName + dataLabelsDetectorDirection(detectorDim)).c_str(),
-                            "unitSI",
-                            &factorDirection);
-
-                        /* position */
-                        std::vector<float_X> positionMesh(simDim, 0.0); /* there is no offset - zero */
-                        hdf5DataFile.writeAttribute(
-                            currentStep,
-                            splashFloatXType,
-                            (meshesPathName + dataLabelsDetectorDirection(detectorDim)).c_str(),
-                            "position",
-                            1u,
-                            splash::Dimensions(simDim, 0, 0),
-                            &(*positionMesh.begin()));
+                            // select data
+                            for(int copyIndex = 0; copyIndex < N_tmpBuffer; ++copyIndex)
+                            {
+                                tmpBuffer[copyIndex]
+                                    = ((picongpu::float_64*)
+                                           values.data())[ampIndex + Amplitude::numComponents * copyIndex];
+                            }
+                            // write actual data
+                            mesh_amp[dir].storeChunk(tmpBuffer, offset_amp, extent_amp);
+                            openPMDdataFile.flush();
+                        }
                     }
+                    // end: write amplitude data
+
+
+                    // start: write observer
+                    // prepare n meshes
+                    ::openPMD::Mesh mesh_n = openPMDdataFileIteration.meshes[dataLabelsDetectorDirection(-1)];
+                    // write mesh attributes
+                    mesh_n.setGeometry(::openPMD::Mesh::Geometry::cartesian); // set be default
+                    // mesh_n.setGeometryParameters( geometryParameters );
```

Comment:
please remove this dead code

Reply:
Sorry @psychocoderHPC for missing this - I will fix it now.

Reply:
@psychocoderHPC I think the first remark originates from a visualization issue in github - the code was already fixed (please see below). The second remark has been fixed. Feel free to review again.